content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -245,7 +243,21 @@ $ export GATEWAY_SERVICE_ACCOUNT=ns/istio-system/sa/bookinfo-gateway-istio
 
 ## Adding your application to the ambient mesh {#addtoambient}
 
-You can enable all pods in a given namespace to be part of an ambient mesh
+When an application pod is part of an ambient mesh, you can check the ztunnel proxy logs to confirm the mesh is redirecting traffic.
+Before we label the namespace to be part of an ambient mesh, check the ztunnel logs related to `inpod` which indicate that in-pod redirection mode is enabled:
```

Comment:
What exactly in the output should the user look for to check that in-pod redirection mode is enabled? `inpod_enabled: true`? The doc test should also check for the same thing.

Reply:
I agree, I thought this is what I was doing exactly for that before and after labeling the ns on my snips:
```
snip_adding_your_application_to_the_ambient_mesh_1() {
kubectl logs ds/ztunnel -n istio-system  | grep inpod
}

! IFS=$'\n' read -r -d '' snip_adding_your_application_to_the_ambient_mesh_1_out <<\ENDSNIP
Found 3 pods, using pod/ztunnel-jrxln
inpod_enabled: true
inpod_uds: /var/run/ztunnel/ztunnel.sock
inpod_port_reuse: true
inpod_mark: 1337
2024-03-26T00:02:06.161802Z  INFO ztunnel::inpod::workloadmanager: handling new stream
2024-03-26T00:02:06.162099Z  INFO ztunnel::inpod::statemanager: pod received snapshot sent
ENDSNIP

snip_adding_your_application_to_the_ambient_mesh_2() {
kubectl label namespace default istio.io/dataplane-mode=ambient
}
```

Do you mean like actually also checking after is labeled? because that is exactly I understood from you before that we didn't need to check so I added this to the one after labeling:

```
{{< text syntax=bash snip_id=none >}}
$ kubectl logs ds/ztunnel -n istio-system  | grep inpod
Found 3 pods, using pod/ztunnel-jrxln
inpod_enabled: true
inpod_uds: /var/run/ztunnel/ztunnel.sock
inpod_port_reuse: true
inpod_mark: 1337
2024-03-26T00:02:06.161802Z  INFO ztunnel::inpod::workloadmanager: handling new stream
2024-03-26T00:02:06.162099Z  INFO ztunnel::inpod::statemanager: pod received snapshot sent
2024-03-26T00:41:05.518194Z  INFO ztunnel::inpod::statemanager: pod WorkloadUid("7ef61e18-725a-4726-84fa-05fc2a440879") received netns, starting proxy
{{< /text >}}
```

now that I think about it, on the before labeling I should just verify for `inpod_enabled: true` and after labeling we just care about `received netns, starting proxy` what do you think?

Reply:
`inpod_enabled` will always be true with our charts - we offer no way to turn it off, it is the only mode. So people should not need to check this.

Reply:
ha, thanks @bleggett I didn't know that. The intention here was to show the before and after labeling logs to show some action without having to get into the actual mechanics as this is intended to be a getting started guide.

As this validation is being done before the ns labeling, do you think it would be better just to check for the string `ztunnel::inpod`?

Reply:
There's no other mode and I don't want to teach people the term `inpod` - 

the checks are "is ambient enabled/do you have a ztunnel, did you label something". There are no other checks people using our charts need.

Reply:
I will open another PR for this, thanks